ci/glci/model.py

```diff
@@ -411,9 +418,26 @@ class AzurePublishCfg:
     storage_account_cfg_name: str
 
 
+@dataclasses.dataclass(frozen=True)
+class OpenRC:
```

Comment:
what's an OpenRC?

Reply:
Hmm openrc are usually openstack enviroment variable files are called. Maybe we rename that e.g. to `OpenstackEnviroment`.

Reply:
I renamed it to `OpenstackEnviroment`